spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'spec_helper'
+
+describe FacilityFactory do
+    before do
+        @facility_factory = FacilityFactory.new
+        @co_sample =
```

Comment:
Using test data like this is a great way to control the data we're getting (rather than calling the API and just using its response) so that we can make more specific assertions. However, this test file then becomes VERY long. Instead of defining the data here, you could use `File.read()` and put this data into another file. We'll learn more about reading from other files in our group project so you'll have a chance to practice this.